src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -413,8 +417,12 @@ public void shouldNotSendPatronNoticeWhenCheckInNoticeIsDefinedAndCheckInDoesNot
     assertThat("Response should not include a loan",
       checkInResponse.getJson().containsKey("loan"), is(false));
 
-    List<JsonObject> sentNotices = patronNoticesClient.getAll();
-    MatcherAssert.assertThat("Notice shouldn't be sent second time",
-      sentNotices.size(), Matchers.lessThan(2));
+    Awaitility.await()
+      .timeout(1, TimeUnit.SECONDS)
```

Comment:
If we believe that it might take up to 5 seconds for notices to appear, why are we only waiting for 1 second to check for the absence of them?

Reply:
Yes, you are correct, replaced this with _pollDelay_ method. Do you think to wait 1 second is ok?